lib/clowne/adapters/sequel.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Clowne
+  module Adapters
+    # Cloning adapter for Sequel
+    class Sequel < Base
+      # Adds #cloner_class method to Sequel::Model
+      module SequelExt
+        def cloner_class
+          return @_clowne_cloner if instance_variable_defined?(:@_clowne_cloner)
+
+          cloner = "#{name}Cloner".safe_constantize
```

Comment:
We cannot use `safe_constantize` here, 'cause it's from ActiveSupport and Sequel doesn't depend on it.

We already have `StringConstantize` refinement with `#constantize` method. We can enhance and use it.

Btw, maybe, we can share this part with AR? It looks identical.

Reply:
Damn, you're right! 
Fixed and implemented ORM extension https://github.com/palkan/clowne/pull/4/commits/db9dd4c6bbb7cb02eb2d9acd165192be48201306